src/site/antora/modules/ROOT/pages/artifacts.adoc

```diff
@@ -0,0 +1,585 @@
+////
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+////
+
+= Artifact reference
+
+The Log4j 2 distribution contains the following artifacts:
+
+[#log4j-bom]
+== `log4j-bom`
+
+A public Bill-of-Materials that manages all the versions of Log4j artifacts.
+You can import the BOM in your build tool of preference:
+
+include::partial$artifacts/log4j-bom.adoc[]
+
+[#log4j]
+== `log4j`
+
+A private Bill-of-Materials used during the compilation and testing of the project.
+
+[CAUTION]
+====
+Do not use this artifact, since it also manages versions of **third-party** projects.
+Use <<log4j-bom>> instead.
+====
+
```

Comment:
Maybe I interpret it incorrectly, but from the fact that we lift `o.a.l.l:log4j` means that people use it in their POMs. I also remember an SO question, where a user was using `log4j` instead of `log4j-bom`.

Since we don't give the same quality guarantees regarding `log4j` as we do with `log4j-bom` (we don't hand-pick the best versions of those dependencies), I would prefer for users not to employ this artifact.

Reply:
Completely agreed, hence my request for removing the section. But anyway...